Assets/Scripts/Models/Tile.cs

```diff
@@ -112,15 +112,14 @@ public float PathfindingCost
 
     public int Y { get; private set; }
 
-    public float MovementModifier;
+    public float MovementModifier { get; private set; }
```

Comment:
I think we need a system of setting conditions or statuses instead of directly modifying the movementmodifier, but I'm changing it back to public until we have that in place.